src/SFA.DAS.ApplyService.Web/Controllers/ApplicationController.cs

```diff
@@ -58,48 +58,44 @@ public async Task<IActionResult> Applications()
 
             if (!applications.Any())
             {
+                //ON-2068 Registered org  with no application created via digital service then
```

Comment:
I don't think we should be adding the ON number to the comments as this should be traceable to the ticket which should have enough details to find out why this is being done - also the title of your pull request does not match this ticket number

Reply:
For my knowledge does Created by the digital service align with org.RoEPAOApproved - if you create this some other way will it not have that flag?

Reply:
As far as I know the all registered orgs will have this flag set. So if there is no applications for a registered org then the org is assummed to be created out side of apply.

Reply:
By putting ON numbers by fixes helps future developers to trace why this change was made especially the branch is merged with master which may have many other changes.